wbooks/src/components/book-detail/book-comments/add-comment/AddComment.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+import styles from './AddComment.styles.scss';
+import perfilExample from '../../../../images/perfilexample.jpg';
+
+class AddComment extends Component {
+  render() {
+    return (
+      <div className={styles.container}>
+        <img
+          className={styles.image}
+          alt = 'perfil'
+          src = {perfilExample}
+        />
+        <div className={styles.addCommentContainer}>
+          <h2 className={styles.title}> Agregar comentario </h2>
+          <textarea className={styles.textarea} rows='6'></textarea>
+          <button className={styles.button}> Enviar </button>
```

Comment:
Its a good idea move Strings to a string file (in the same folder that each component) and import them here.
This is because on each render you are generating a new string instance! instead of use the string from a consts file taht will be just a reference, saving memory each render 😄